src/test/run-pass/const-int-sign.rs

```diff
@@ -1,11 +1,19 @@
 const NEGATIVE_A: bool = (-10i32).is_negative();
```

Comment:
I think these test files now need `#![feature(const_int_sign)]`

Reply:
Indeed. Sorry, I was relying on CI to run the tests.